README.md

```diff
@@ -90,7 +90,7 @@ image we provide:
 docker run
   -e MARGE_AUTH_TOKEN="$(cat marge-bot.token)" \
   -e MARGE_SSH_KEY="$(cat marge-bot-ssh-key)" \
-  smarketshq/marge-bot:0.1.0 \
+  smarketshq/marge-bot \
```

Comment:
Don't think this has the right semantics, unless you bump latest as well (just creating an new tag w/ make dockerize won't, I think).